.github/workflows/release.yml

```diff
@@ -86,6 +86,10 @@ jobs:
           DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
       - name: build-amd64
         run: "docker buildx build --platform linux/amd64 -t ${{ github.repository }}:${{ env.VERSION }} --push --cache-from type=registry,ref=${{ github.repository }} --cache-to type=registry,ref=${{ github.repository }},mode=max ."
+      - name: buildkit-amd64
```

Comment:
buildkit requires an specific image and come in two "flavors", "privileged" and "rootless". The preferred flavor should be "rootless" but there are some k8s environments that do not work with it, so we use the "privileged" version